src/Language/PureScript/Docs/RenderedCode/RenderType.hs

```diff
@@ -154,12 +154,6 @@ insertPlaceholders :: RenderTypeOptions -> Type -> Type
 insertPlaceholders opts =
   everywhereOnTypesTopDown convertForAlls . everywhereOnTypes (convert opts)
 
-dePrim :: Type -> Type
-dePrim ty@(TypeConstructor (Qualified _ name))
-  | ty == tyBoolean || ty == tyNumber || ty == tyString =
-    TypeConstructor $ Qualified Nothing name
-dePrim other = other
```

Comment:
Ah ok, that totally makes sense! It was probably in here originally for `psc-docs`.